How to Find and Use Technical Resources if You’re New to Tech:proposal.md

```diff
@@ -0,0 +1,60 @@
+
+**Taking coding lessons with freeCodeCamp**
+ 
+ One great place for you to start as a newbie to coding is [freeCodeCamp](https://www.freecodecamp.org/). This nonprofit organization helps users code through hands-on projects and structured, easy-to-manage lessons. 
+
+In addition to their learning platform, they also have Youtube videos and can connect you to a community of others who are just as excited about coding as you are. 
+
+If you're feeling particularly ambitious, they offer certification programs too in topics like responsive web design and Javascript algorithims and data structures. 
+  
+**Putting your new coding skills to work**
+
+So you've built up some coding skills. Where do you go from here? Now you can begin cool projects that you get to show off later on. These can be as simple as a website portolio that you code from scratch — maybe in a code editor like [CodePen](https://codepen.io/) — or getting a little more complicated like an app that chooses what you eat for dinner. 
+
+Your projects are only limited by your imagination, and even if you're feeling a bit stuck, you can seek out inspiration from others online too. 
+
+**Using W3Schools for coding references** 
+
+Sometimes if you need other examples of coding while you're working on your project, [W3Schools](https://www.w3schools.com/) is a good place to go.They only offer written tutorials, but it's a good reference point. 
+
+Here, you can sort through examples they provide in the coding language you are using to see where your code might be causing problems. You can even open the code editor for each example and try playing around to  gain a better understanding of the concept you're trying to grasp. 
+
+They offer lots of examples in many different languages, so hopefuly it can be helpful to you no matter what you're coding in. 
+
+**Finding more coding resources**
+
+The few resources I listed above may get you started on learning coding, but don't think that your list of options ends there. Search engines are your best friends here. 
+
+You can look up more detailed walkthroughs of coding concepts in Youtube tutorials, or you can find other coding lessons for free in many many other sites like [Khanh Academy](https://www.khanacademy.org/computing/computer-programming). 
+
+Sometimes you have to get specific with what you're searching to get faster, better results. If you get stuck, it can be be frustrating. But don't give up! The internet gives you plenty of free resources at your fingertips within seconds.
+  
+Now that you know a couple of ways to navigate the internet for coding resources, the possibilites are endless. You can learn as little or as much code as you want, all without paying for expensive classes. Use what you learn to start your own project — perhaps a website or an interactive fiction — and share what you create with others. 
```

Comment:
I think you could also expand on the common mistakes programmers can make, so you could then describe other resources that help programmers improve. This could be done when interviewing other programmers. I also think linking some projects made by other Bit Project programmers would be great too.